### PR TITLE
Updating copy per blog post feedback

### PR DESCRIPTION
**Pull request by katierose4**
@mlettini Updated the copy on [our notable page](http://harvesthq.github.io/notable/) per the blog post feedback. Let me know if this looks good. Thanks!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -35,13 +35,12 @@ <h2 id="project_tagline">A quick way to get important messages to your team</h2>
...
+        <p>How do we communicate important announcements to our 45 person team across nine different countries here at Harvest? We use a single email to aggregate all of the team’s notable messages for the day. It’s the one email that everyone on the team reads every day. It highlights notable things like:</p>
         <ul>
           <li><em>Hey team, we shipped this new feature today. It’s out to 75% of customers.</em></li>
-          <li><em>We’re hiring! If you know any developers, have them apply here.</em></li>
-          <li><em>Eike and his wife had a baby boy!</em></li>
+          <li><em>Reminder that our Summit is in NYC next week. Please remember to bring a gift for the gift exchange!</em></li>
         </ul>
-        <p>Isn’t this what email is for? Not quite. Emails are a bit stuffy and take time to put together. Slack, on the other hand, is much more informal and a quicker way to get a message out. That’s why we’ve chosen it as our channel to make important, company-wide announcements.</p>
+        <p>Thie email highlights notable, company-wide announcements so that they don’t get lost amid GIFs and emojis in Slack.</p>
```
Comment (mlettini) [inline index.html]: Spelling on "This" is off

File: index.html [hunk 2]
```diff
@@ -35,13 +35,12 @@ <h2 id="project_tagline">A quick way to get important messages to your team</h2>
...
-        <p>Our Slack channels are fairly active at Harvest. GIFs are flying everywhere, people are asking what’s for lunch at all times of the day, and cryptic, random conversations are taking place in the #emoji-only channel. But, mixed in among all the madness, there are actually valid, important things that everyone in the company should know, like:</p>
+        <p>How do we communicate important announcements to our 45 person team across nine different countries here at Harvest? We use a single email to aggregate all of the team’s notable messages for the day. It’s the one email that everyone on the team reads every day. It highlights notable things like:</p>
         <ul>
           <li><em>Hey team, we shipped this new feature today. It’s out to 75% of customers.</em></li>
-          <li><em>We’re hiring! If you know any developers, have them apply here.</em></li>
-          <li><em>Eike and his wife had a baby boy!</em></li>
```
Comment (mlettini) [inline index.html]: I like this one - it's a real human element to this feature. Leave it in!

**Comment by mlettini** [comment]
two comments

**Comment by katierose4** [comment]
Updated! How about now @mlettini?

**Comment by mlettini** [comment]
+1

**Pull request merged**